src/redteam/index.ts

```diff
@@ -252,16 +255,46 @@ export async function synthesize({
       );
       logger.debug(`Added ${pluginTests.length} ${plugin.id} test cases`);
       pluginResults[plugin.id] = { requested: plugin.numTests, generated: pluginTests.length };
+    } else if (plugin.id.startsWith('file://')) {
+      logger.debug(`Loading custom plugin from ${plugin.id}`);
+      if (plugin.id.endsWith('.yaml')) {
+        const filePath = plugin.id.slice('file://'.length);
+
+        const definition = yaml.load(fs.readFileSync(filePath, 'utf8')) as {
```

Comment:
nit, consider using maybeLoadFromExternalFile. It does some nice path manipulation with cliState.basePath which is important if we want to support relative paths

Reply:
great idea